options/ansi/include/setjmp.h

```diff
@@ -13,8 +14,16 @@ extern "C" {
 // FIXME: use intptr_t equivalent
 typedef long jmp_buf[__MLIBC_JMPBUF_SIZE];
 
+typedef struct sigjmp_buf {
+	jmp_buf __jmpbuf;
+	int __mask_was_saved;
+	sigset_t __saved_mask;
+} sigjmp_buf;
```

Comment:
The standard defines `jmp_buf` to be an array. Is `sigjump_buf` supposed to be a struct?

Reply:
POSIX says:
> The <setjmp.h> header shall define the array types jmp_buf and sigjmp_buf.